packages/create-remix/templates/fly-stack/prisma/schema.prisma

```diff
@@ -8,16 +8,30 @@ generator client {
 }
 
 model User {
-  id       String @id @default(cuid())
-  email    String @unique
-  password String
+  id    String @id @default(cuid())
+  email String @unique
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  password Password?
```

Comment:
This should not be optional right?

```suggestion
  password Password
```

Reply:
yes! but, Prisma won't allow that 

> The relation field `password` on Model `User` is required. This is no longer valid because it's not possible to enforce this constraint on the database level. Please change the field type from `Password` to `Password?` to fix this.

Reply:
we might be able to make a [composite id](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#composite-ids), but i think that requires at least 2 properties to be on a model, we just have "hash" on it